<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>User</class>
 <widget class="QWidget" name="User">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>764</width>
    <height>504</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>User</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <layout class="QHBoxLayout" name="horizontalLayout_4">
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_3">
       <item>
        <layout class="QVBoxLayout" name="verticalLayout_2">
         <item>
          <widget class="QLabel" name="eventslabel">
           <property name="font">
            <font>
             <family>Verdana</family>
             <pointsize>12</pointsize>
            </font>
           </property>
           <property name="text">
            <string>All Events</string>
           </property>
           <property name="buddy">
            <cstring>events</cstring>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QListWidget" name="events">
           <property name="font">
            <font>
             <family>Verdana</family>
             <pointsize>12</pointsize>
            </font>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QVBoxLayout" name="verticalLayout">
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout">
           <item>
            <widget class="QLabel" name="monthLabel">
             <property name="font">
              <font>
               <family>Verdana</family>
               <pointsize>12</pointsize>
              </font>
             </property>
             <property name="text">
              <string>Month</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLineEdit" name="monthEdit">
             <property name="font">
              <font>
               <family>Verdana</family>
               <pointsize>12</pointsize>
              </font>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="filterButton">
             <property name="font">
              <font>
               <family>Verdana</family>
               <pointsize>12</pointsize>
              </font>
             </property>
             <property name="text">
              <string>see</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_2">
           <item>
            <widget class="QPushButton" name="nextButton">
             <property name="font">
              <font>
               <family>Verdana</family>
               <pointsize>12</pointsize>
              </font>
             </property>
             <property name="text">
              <string>Next</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="addButton">
             <property name="font">
              <font>
               <family>Verdana</family>
               <pointsize>12</pointsize>
              </font>
             </property>
             <property name="text">
              <string>Add</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_3">
           <item>
            <widget class="QPushButton" name="deleteButton">
             <property name="font">
              <font>
               <family>Verdana</family>
               <pointsize>12</pointsize>
              </font>
             </property>
             <property name="text">
              <string>Delete</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="saveButton">
             <property name="font">
              <font>
               <family>Verdana</family>
               <pointsize>12</pointsize>
              </font>
             </property>
             <property name="text">
              <string>Save</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="fileButton">
             <property name="font">
              <font>
               <family>Verdana</family>
               <pointsize>12</pointsize>
              </font>
             </property>
             <property name="text">
              <string>Open in file</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
        </layout>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_4">
       <item>
        <widget class="QLabel" name="listlabel">
         <property name="font">
          <font>
           <family>Verdana</family>
           <pointsize>12</pointsize>
          </font>
         </property>
         <property name="text">
          <string>List of events</string>
         </property>
         <property name="buddy">
          <cstring>list</cstring>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QListWidget" name="list"/>
       </item>
      </layout>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
